ksp_plugin/plugin.cpp

```diff
@@ -418,9 +418,8 @@ void Plugin::InsertOrKeepLoadedPart(
     }
   } else {
     Instant const previous_time = current_time_ - Δt;
-    enum class LocalTag { tag };
-    using MainBodyCentred =
-        geometry::Frame<LocalTag, LocalTag::tag, /*frame_is_inertial=*/false>;
+    auto const Δplanetarium_rotation =
+        Exp(Δt * angular_velocity_of_world_).Forget();
     BodyCentredNonRotatingDynamicFrame<Barycentric, MainBodyCentred> const
```

Comment:
I wonder if this computation could be factored with `BarycentricToWorld`.  Put a TODO to investigate.

Reply:
Done.

Reply:
Added an enum.